examples/plot_afq_api.py

```diff
@@ -119,7 +119,7 @@
 # We will then use `nibabel` to load the deriviative file and retrieve the
 # data array.
 
-FA_fname = myafq.dti_fa[0]
+FA_fname = myafq.dti_fa["01"]
```

Comment:
Is this the subject ID? If there is more than one session would this be as follows? 

```myafq.dti_fa["01"]```

Reply:
Meant to say - would it be:

```myafq.dti_fa["01"]["01"]```

Reply:
Yes. If there was more than one session, and a session name is pre, then it would be:
`myafq.dti_fa["01"] ["pre"]`